src-docs/src/views/button/button_empty.js

```diff
@@ -6,36 +6,52 @@ import {
   EuiFlexItem,
 } from '../../../../src/components/';
 
+const buttons = ['primary', 'success', 'warning', 'danger', 'text', 'disabled'];
+
 export default () => (
   <div>
-    <EuiFlexGroup gutterSize="s" alignItems="center">
-      <EuiFlexItem grow={false}>
-        <EuiButtonEmpty onClick={() => {}}>Primary</EuiButtonEmpty>
-      </EuiFlexItem>
-
-      <EuiFlexItem grow={false}>
-        <EuiButtonEmpty size="s" onClick={() => {}}>
-          small
-        </EuiButtonEmpty>
-      </EuiFlexItem>
-
-      <EuiFlexItem grow={false}>
-        <EuiButtonEmpty size="xs" onClick={() => {}}>
-          extra small
-        </EuiButtonEmpty>
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    {buttons.map((value) => (
+      <>
+        <EuiFlexGroup gutterSize="s" key={value} alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              style={{
+                textTransform: 'capitalize',
+              }}
+              isDisabled={value === 'disabled' ? true : false}
+              color={value !== 'disabled' ? value : 'primary'}
+              onClick={() => {}}>
+              {value}
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              isDisabled={value === 'disabled' ? true : false}
+              color={value !== 'disabled' ? value : 'primary'}
+              size="s"
+              onClick={() => {}}>
+              small
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              isDisabled={value === 'disabled' ? true : false}
+              color={value !== 'disabled' ? value : 'primary'}
+              size="xs"
+              onClick={() => {}}>
+              extra small
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </>
+    ))}
 
     <EuiFlexGroup gutterSize="s" alignItems="center">
       <EuiFlexItem grow={false}>
         <EuiButtonEmpty onClick={() => {}} iconType="arrowDown">
-          Primary
-        </EuiButtonEmpty>
-      </EuiFlexItem>
-
-      <EuiFlexItem grow={false}>
-        <EuiButtonEmpty size="s" onClick={() => {}} iconType="arrowDown">
-          small
+          arrowDown
```

Comment:
```suggestion
          Icon left
```

Mainly because starting the button label with a lowercase letter is awkward.